app/jobs/clock/finalize_invoices_job.rb

```diff
@@ -9,7 +9,7 @@ class FinalizeInvoicesJob < ApplicationJob
     unique :until_executed, on_conflict: :log
 
     def perform
-      Invoice.ready_to_be_finalized.find_each do |invoice|
+      Invoice.ready_to_be_finalized.includes(:credit_notes).find_each do |invoice|
```

Comment:
How does this include help? 🤔

Reply:
inside `FinlizeJob` we pass this invoice to `Invoices::RefreshDraftAndFinalizeService`, where there we're passing it further to `RefreshDraftService`, where finally we're going through 
```
        cn_subscription_ids = invoice.credit_notes.map do |cn|
          {credit_note_id: cn.id, subscription_id: cn.fees.pick(:subscription_id)}
        end
```
and actually in addition to the credit notes we can also preload fees... 🤔

Reply:
but the inclusion here does nothing right? we load the invoice again in that job?